english/meditate/meditate.md

```diff
@@ -1,4 +1,4 @@
-You recently read that meditation helps you feel more awake, relieves stress, and helps you prepare for the day ahead.
+You recently read that meditation helps you feel more awake, relieves stress, attracts hippies, and helps you prepare for the day ahead.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.